### DISCUSSION TITLE
CrazyFlie Lighthouse Positioning System- Unable to receive sweeping angles

### DISCUSSION THREAD

**Discussion by StudentE019** [bitcraze/discussions#1163] [Q&A]
![Screenshot from 2024-01-31 14-53-54](https://github.com/bitcraze/discussions/assets/158256500/67f18348-bb00-4065-8595-010db914baea)
Apparently the receiving and calibration end of the two V2 base station lighthouses, are marked green( meaning they are connected) but no sweeping angles are detected by the base stations. Is there a way to solve this?

**Comment by knmcguire** [comment]
It is strange that you do seem to receive the lighthouse base stations. I'm assuming that you were following the steps of the tutorial before starting with this wizard right? https://www.bitcraze.io/documentation/tutorials/getting-started-with-lighthouse/?

**Comment by StudentE019** [comment]
Yes. All the instructions were followed but when I clicked on the 'Manage Geometry' Tab and subsequently the 'Estimate Geometry' tab and went on to start measurements, it just responded by sending 'No sweeping angles'. I am not sure what that means.
________________________________

**Reply by knmcguire** [reply]
that is strange. I have some additional questions.

- On which OS are you running the client on?
- In the console tab, do you see any errors? Could you copy-paste the content?
- In the plotter tab, could you check if it detects the lighthouse angles from these logging variables?: https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/logs/#lighthouse

**Comment by StudentE019** [comment]
Ubuntu was being used. The error shown is in the image below.[https://res.cdn.office.net/assets/mail/file-icon/png/photo_16x16.png]photo1706688513.jpeg<https://entuedu-my.sharepoint.com/:i:/g/personal/ar0001ed_e_ntu_edu_sg/EYiYrEhtpl1Gla9AbYD0XfMBJdp11dgjrWqJpN6snNjjaw>

________________________________

**Reply by knmcguire** [reply]
could you upload the picture on the website gui of discussions? I can not see the picture here

**Comment by StudentE019** [comment]
Unfortunately, when I input my username to login to the forum it does not recognize my Username. The image containing the error is below.[cid:4a973307-4d36-4dbd-ae9c-06ae719df22d]
________________________________

**Reply by knmcguire** [reply]
Sorry, the image can still not be shown. This discussion platform is part of Github so please check how to retrieve your Github account credentials. 

If this all doesn't work you can also send us and email at contact _at_ bitcraze.io but mind that we are must faster in responding here

**Reply by abhi9649402310** [reply]
mam i'm also facing same issue and I mailed you but regarding that they send me to this discussion but I'm not able to find any useful information to solve my problem

**Reply by knmcguire** [reply]
@abhi9649402310  welcome!

If you can't find an answer here then it is best that you start a new discussion thread and we can help you on your specific case. We prefer working in the open platforms like this such that our answers can be used for others. In this case, our help for you will help others as well.
